### PR TITLE
[pl] updating the remaining dictionary entries

### PR DESCRIPTION
**Pull request by makowskid**
100% now

### CODE REVIEW

File: docs/referents.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ pl
 
 </td><td width="50%">
 
-@Olsza
+@Olsza, @makowskid
```
Comment (andrey-helldar) [inline docs/referents.md:100]: Do you want to join our team?

**Comment by andrey-helldar** [comment]
Ignore the tests, this is an internal error. I'll fix it later.

**Review by andrey-helldar — APPROVED** [review]
Dziękuję

**Comment by andrey-helldar** [comment]
Do you want to join our team?

Each of us is engaged in the fact that he monitors the chosen localization. When you need to fix something or new keys have been added - translate them. When someone creates a PR for this localization, check it out.

Usually when someone submits a localization PR, @caouecs or I assign it to the users to check against the [referent](https://laravel-lang.com/referents.html) table.

Interesting?

**Comment by makowskid** [comment]
Sure, I'd love to help. Please assign PRs/issues to me as you like.
You can include me in your other projects as well for translations.

**Comment by andrey-helldar** [comment]
@caouecs, hey! Can you add a new member in our team? 🙂

**Comment by andrey-helldar** [comment]
@makowskid, we have three projects with translations:

- [Laravel-Lang/lang](https://github.com/Laravel-Lang/lang)
- [Laravel-Lang/attributes](https://github.com/Laravel-Lang/attributes)
- [Laravel-Lang/http-statuses](https://github.com/Laravel-Lang/http-statuses)

**Comment by caouecs** [comment]
done

**Comment by makowskid** [comment]
I will review and monitor it. Please ping me if necessary.

**Comment by andrey-helldar** [comment]
> I will review and monitor it. Please ping me if necessary.

Okay

**Comment by makowskid** [comment]
looking into it ;-)

**Pull request merged**